tests/testExecutables.py

```diff
@@ -0,0 +1,12 @@
+import unittest
+import lsst.utils.tests
+
+
+class UtilsBinaryTester(lsst.utils.tests.ExecutablesTestCase):
+    pass
+
+EXECUTABLES = None
+UtilsBinaryTester.create_executable_tests(__file__, EXECUTABLES)
```

Comment:
Yes it does. Thanks. The reason it is there is that this is present in the template version of this test file from the `utils` package. I did want the template to include the tuple so that people would understand that they have the option of including an explicit list in their code.